crates/blockchain/Cargo.toml

```
@@ -28,6 +28,7 @@ path = "./blockchain.rs"
 
 [features]
 default = ["c-kzg"]
+default-levm = ["default", "levm"]
```

Let's name the target to `levm`
```suggestion
levm = ["default", "levm"]
```

---

There is already a levm target
`levm = ["ethrex-vm/levm"]`
Do you think I should just do: `levm = ["default", "ethrex-vm/levm"]`?

---

Yes!

---

[Done](https://github.com/lambdaclass/ethrex/pull/1546/commits/8fa56fdff2db8f4edc9faa3f39eca9025ab1e6ad)